src/features/member/tabs/campaigns-tab/index.tsx

```diff
@@ -40,11 +61,17 @@ export const CampaignsTab: React.FC<{
         </RefreshButton>
       </MainHeadline>
       <CampaignsInfo
+        focused={navigationStep + 1 === 0}
+        navigationAvailable={focus === FocusItems.Member.items.CampaignsRedeem}
         memberId={memberId}
         referralInformation={referralInformation}
       />
 
-      <ReferralsInfo referralInformation={referralInformation} />
+      <ReferralsInfo
+        referralInformation={referralInformation}
+        navStep={navigationStep}
```

Comment:
Preferably no shorthands
```suggestion
        navigationStep={navigationStep}
```